docs/install/source-step-by-step.md

```diff
@@ -0,0 +1,336 @@
+---
+html_meta:
+  "description": "Install Plone 6 backend from scratch for the one who wants to look under the hood"
+  "property=og:description": "Install Plone 6 backend from scratch for the one who wants to look under the hood"
+  "property=og:title": "Install Plone backend from Scratch – Step by Step"
+  "keywords": "Plone, Plone 6, install, backend, pip, mxdev, mxmake, cookiecutter, source, Zope, buildout"
+---
+
+
+(install-source-stepbystep-start-label)=
+
+# Install Plone backend from Scratch – Step by Step
+
+
+(install-source-stepbystep-backend-start-label)=
+
+## Backend
+
+For system requirements and pre-requisites for the installation see {ref}`install-source-system-requirements-label`.
+
+We install the Plone backend with `pip`, `cookiecutter-zope-instance`, `mxdev` and other fancy helpers.
+
+```{note}
+There will be one single cookiecutter template to install both backend and frontend from scratch. You will find the instructions on {ref}`install-source-installation-jump-label`. That chapter is for you if you want to develop and want to jump in with all steps prepared by an overall cookiecutter. The subsequent sections explain the installation of the backend step by step. You will learn the details of the installation included in the future overall cookiecutter.
+```
+
+
+(install-source-installation-steps-label)=
+
+### Installation steps
+
+Create a new directory to hold your project, make it your current directory, then issue the following commands in a shell session.
+
+Create a Python virtual environment in the current directory.
+
+```shell
+python3.9 -m venv venv
+source venv/bin/activate
+```
+
+Update Python package management tools.
+
+```shell
+pip install -U pip wheel
+```
+
+````{admonition} packages tree
+:class: margin toggle
+The Plone packages and dependencies are installed in trees.
+
+% TODO Why in trees?
+
+```
+venv/lib/Python3.x/
+site-packages/
+┌── plone/
+│   ├── …
+│   ├── app/
+│       ├── caching/
+│       ├── content/
+│       └── …
+…   
+├── zope/
+│   ├── …
+```
+````
+
+Install Plone 6 with constrained requirements using `pip`.
+
+```shell
+pip install Plone -c https://dist.plone.org/release/6.0.0a4/constraints.txt
+```
+
+````{admonition} mkwsgiinstance's minimal Zope configuration
+:class: margin toggle
+`mkwsgiinstance` creates a home with a minimal configuration for a Zope instance.
+No blob storage, etc. is defined so far.
```

Comment:
I would add here: "Plone starts with this configuration, but you would need to edit the files manually to get things like blobstorage for files and images and possible addons working properly."

Reply:
yes. As this is a sidebar, I even reduce to the skeleton tree without any further text. 
Anything else can be sayd in main content.